src/jsMain/kotlin/net/kyori/adventure/webui/js/Main.kt

```diff
@@ -403,6 +403,8 @@ public fun main() {
                     hoverTooltip.style.left = "${left}px"
                 }
             )
+
+            installStyleButtons()
```

Comment:
Not a criticism but I like this - I wonder if we could migrate more stuff like this?

Reply:
Yep! I had a wip branch which moved the math + stuffs behind hover into a separate function like this